aerogear/graphback
# Remove need for database initialization strategies

## What

- Decouple graphql-migrations from graphback
- Use graphql-migrations in CLI

## Tasks

- [x] Decouple graphql-migrations from graphback
- [x] Use graphql-migrations in CLI
- [x] Remove `graphback update-db` command
- [ ] Use `graphql-config` to load schema

## Verification

**Ignore for now. This is a draft PR**.

1. Run `examples/runtime-example`. Everything should work as before (migration is created).
2. Run `graphback db`. It should use `graphql-migrations` to create migration metadata.

## Other

This is a draft PR.
